.github/workflows/build-fb-image.yaml

```diff
@@ -36,18 +45,100 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
+      
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
 
       - name: Login to Docker Hub
         uses: docker/login-action@v3
         with:
+          registry: docker.io
           username: ${{ secrets.REGISTRY_USER }}
           password: ${{ secrets.REGISTRY_PASSWORD }}
 
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@v3
-
+      
       - name: Build and Push Image for Fluent Bit
-        run: |
-          make build-fb -e FB_IMG=${{ env.FB_IMG }}
-          make build-fb-debug -e FB_IMG_DEBUG=${{ env.FB_IMG_DEBUG }}
\ No newline at end of file
+        id: docker-build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./cmd/fluent-watcher/fluentbit/Dockerfile
+          push: true
+          platforms: linux/amd64,linux/arm64
+          tags: ${{ env.FB_IMG}}
+      
+      - name: Build and Push Debug Image for Fluent Bit
+        id: docker-build-debug
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./cmd/fluent-watcher/fluentbit/Dockerfile.debug
+          push: true
+          platforms: linux/amd64,linux/arm64
+          tags: ${{ env.FB_IMG_DEBUG }}
+        
+  scan-docker-image:
+    name: Scan Docker Image
+    needs: 
+      - build
+    uses: ./.github/workflows/scan-docker-image-action.yaml
+    with:
+      source_image: ${{ needs.build.outputs.FB_IMG }}
+      source_registry: docker.io
+      platforms: '["linux/arm64", "linux/amd64"]'
+    secrets:
+      registry_username: ${{ secrets.REGISTRY_USER }}
+      registry_password: ${{ secrets.REGISTRY_PASSWORD }}
+  
+  release-image-to-gchr:
+    name: Release Image to GitHub Container Registry
+    uses: ./.github/workflows/clone-docker-image-action.yaml
+    needs: 
+      - scan-docker-image
+      - build
+    with:
+      source_image: ${{ needs.build.outputs.FB_IMG }}
+      source_registry: docker.io
+      target_image: ${{ needs.build.outputs.FB_IMG_GHCR }}
+      target_registry: ghcr.io
+      platforms: "['linux/arm64', 'linux/amd64']"
+    secrets:
+      source_registry_username:  ${{ secrets.REGISTRY_USER }}
+      target_registry_username: ${{ github.actor }}
+      source_registry_token: ${{ secrets.REGISTRY_PASSWORD }}
+      target_registry_token: ${{ secrets.GITHUB_TOKEN }}
+
+  scan-debug-image:
```

Comment:
The debug image should not be scanned, it should trigger notifications as it'll have a load of extra dependencies and is not intended for production. Any complaints about CVEs on debug images should be ignored.